src/main/java/seedu/address/model/eatery/Eatery.java

```diff
@@ -63,6 +64,20 @@ public Category getCategory() {
         return category;
     }
 
+    public void addReview(Review review) {
+        reviews.add(review);
+    }
+
+    public Set<Review> getReviews() {
+        return reviews;
+    }
+
+    public void setReviews(Set<Review> reviews) {
```

Comment:
This method seems dangerous – What happens if there are existing reviews when this method is called? I assume the desired behavior is to replace the existing reviews with the new set, and not to concatenate them?